mediaplatform_jwp/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+
+class Video(models.Model):
+    """
+    A JWPlatform video resource.
+
+    """
+    #: JWPlatform video key
+    key = models.CharField(primary_key=True, max_length=64, editable=False)
+
+    #: Corresponding :py:class:`mediaplatform.MediaItem`. Accessible from the
+    #: :py:class:`mediaplatform.MediaItem` model via the ``jwp`` field.
+    item = models.OneToOneField('mediaplatform.MediaItem', related_name='jwp',
```

Comment:
same comment as with sms legacy

Reply:
A similar reply to the one about SMS but with the subtlety that it is unclear to me if it is ever valid to have a video on JWPlatform which has no corresponding ``mediaplatform.MediaItem`` entry. I've assumed not but my assumption could be mistaken.